### PR TITLE
bring package dependencies up to date, and use semantic versioning

### PR DESCRIPTION
**Pull request by iislucas**
I'm slightly worried about the old custom github branch - not sure why that was needed? 

But the other changes here are just to use semantic versioning so this doesn't break downstream projects that try to use it.

### CODE REVIEW

**Comment by rachelrosengoogle** [comment]
I think I did the correct angular updates here and just forgot to submit: https://github.com/conversationai/perspectiveapi-authorship-demo/pull/64 Going to try and resolve those, then will look at this. Sorry about that!

**Comment by iislucas** [comment]
that PR correctly specifies angular v6, but incorrectly specifies it w.r.t. semantic versioning. See the edited files of this PR (the ^ added to package versions)

File: package.json [hunk 1]
```diff
@@ -18,20 +18,20 @@
...
+    "@angular/compiler-cli": "^6.0.0",
+    "@angular/core": "^6.0.0",
+    "@angular/elements": "^6.0.0",
+    "@angular/forms": "^6.0.0",
+    "@angular/http": "^6.0.0",
+    "@angular/material": "^6.0.1",
+    "@angular/platform-browser": "^6.0.0",
+    "@angular/platform-browser-dynamic": "^6.0.0",
+    "@angular/router": "^6.0.0",
+    "gsap": "^1.20.4",
```
Comment (rachelrosengoogle) [inline package.json]: We can't use semantic versioning for gsap because 1.20.5 has breaking changes: https://github.com/greensock/GreenSock-JS/issues/266

Comment (iislucas) [reply package.json]: Actually, looks like the API changes are trivial to apply, so I'll just do that and then we don't have to make this horribly inflexible in the peer dependencies :)

**Comment by iislucas** [comment]
I think update now this fixes the issue you raised; PTAL. 

But given tests seem to be broken on master branch, I'm not sure quite how I can tell. I scanned the failing tests and didn't see and gsap import errors -- is that what you were expecting to happen?

**Comment by iislucas** [comment]
Moved this work to the other PRs (https://github.com/conversationai/perspectiveapi-authorship-demo/pull/81 and https://github.com/conversationai/perspectiveapi-authorship-demo/pull/79)

**Pull request closed without merging**